app/jobs/resource_requests/iiif_deregistration_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module ResourceRequests
+  class IiifDeregistrationJob < AbstractJob
+    include ResourceRequestJobs::IiifJobBehaviors
+
+    @queue = :image_syndication
```

Comment:
```suggestion
    @queue = :image_deregistration
```
Suggesting a rename in case this was missed, but also because I think we'll want registration (higher priority) to be on a different queue than deregistration (lower priority).